### DISCUSSION TITLE
Next Release

### DISCUSSION THREAD

**Discussion by lafranst** [Orcpub/orcpub#578] [General]
Hi! 

I was wondering when the next release will be... well... released? 

I am not sure how the developers work this out, being mostly donated time and all. All I can see is a lot of issues/enhancements still unresolved after a lot of time has passed. And please understand that I am NOT complaining here. This is just a statement of fact that I have completely accepted, especially since this is a free tool. Hopefully this project is still alive and well, as I really like what you are doing with this product.

Talking about coding... I used to be a developer in the days of the dinosaurs. I still work in the business but have not coded in quite some time. Is this a difficult environment and language to work with, and should I stay away from even thinking about contributing? I'm asking the mammals that have replaced the dinosaurs... :)

Thanks,

Stephan

**Comment by datdamnzotz** [comment]
I was hoping to get to a new release some time this year but with a move and life it got hectic and I haven't had any time to work on getting a developer shop setup.
As of right now funds are being used to host the site.

**Comment by Jessomadic** [comment]
I would like to work on it and I hope to get something I was working on back at the start of the year out but I can't make an guarantees. I would still consider the project alive and we don't take any of the money for ourselves.

**Comment by codeGlaze** [comment]
@lafranst 

re: trying to hop into the project
It's all written in Clojure which is a modern LISP language, but leverages Java. So if you're accustom to C or C++ you probably have the mindset able to get into it. But there's definitely a learning curve (which we can attempt to help you through).

Clojure is a weird middle ground between dynamic languages and static languages. Definitely... *Interesting* to say the least.

There are complicating factors, as with most things, such as front end logic basically being flavored javascript... but that can be explained in the discord if you'd like.

The couple, or so, of us who have been addressing bugs and looking into enhancements pretty much uniformly have gotten busier since the pandemic hit, so things have certainly slowed. But the site is actively administrated and moderated by our tiny crew.

**Reply by lafranst** [reply]
Wow, is C and C++ considered as being from the Jurassic period (of computer development)? I guess it is... Yes, I am familiar with them, as I've done a few years of both of them. But by dinosaur I meant the Cobol days... But then again, if C is from the Jurassic period, both Cobol and LISP is from the Triassic period, whether or not there are modern versions... lol

I'll take a look at Clojure and see if I will/can still offer some help there.

Thanks all for the info!

**Reply by codeGlaze** [reply]
🤣 C was the oldest 'modern' language I could think of.
COBOL, BASIC, etc are too far from the current design paradigm with their chunky keywords and --*shudder*-- Level Numbers. xD
